src/routers/account.js

```diff
@@ -524,7 +524,8 @@ router.get('/upgrade', async (request, response) => {
 		layout: 'main',
 		locale: util.getLocale(request.locale.region, request.locale.language),
 		localeString: request.locale.toString(),
-		error: request.cookies.error
+		error: request.cookies.error,
+		currentTier: 'price_1LBnZADOJlJAaQQ3pEUjNWbY' // To be replaced
```

Comment:
cant this be implemented now that the other PR was merged?

Reply:
It probably can. psst @jonbarrow

Reply:
@ashmonty @mrjvs I will add that in with my next feature PR, I still need to add the cancellation feature

Reply:
Okie-dokie!